src/api/app/components/notification_notifiable_link_component.rb

```diff
@@ -123,7 +124,7 @@ def notifiable_link_path
     when 'Event::ReportForProject', 'Event::ReportForPackage'
       @notification.event_type.constantize.notification_link_path(@notification)
     when 'Event::ReportForUser'
-      Rails.application.routes.url_helpers.user_path(@notification.event_payload['user_login'], notification_id: @notification.id)
+      Rails.application.routes.url_helpers.user_path(@notification.event_payload['user_login'], notification_id: @notification.id) if User.find_by_login(@notification.event_payload['user_login'])
```

Comment:
How can a user not exist?

Reply:
I tackled the `User.destroy` case, but now after double checking the code, I realized we never do that. We only `delete` it.